sdk/loadtestservice/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Release History
 
+## 1.0.0 
```

Comment:
please add date before merge

Reply:
added date here: https://github.com/Azure/azure-sdk-for-python/pull/27274/commits/c7cbbdaa9b137080e234800a70154c6bb80badf4